packages/world/ts/getWorldAbi.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, expect, it, vi } from "vitest";
+import { getWorldAbi } from "./getWorldAbi";
+import { testClient } from "./common";
+
+vi.mock("./getFunctions", () => {
+  const mockGetFunctionsResult = [{ signature: "setNumber(bool)" }, { signature: "batchCall((bytes32,bytes)[])" }];
+  const getFunctions = vi.fn();
+  getFunctions.mockResolvedValue(mockGetFunctionsResult);
+
+  return {
+    getFunctions,
+  };
+});
+
+describe("World ABI", () => {
+  it("should concat base and world ABI", async () => {
+    const abi = await getWorldAbi({
+      client: testClient,
```

Comment:
if `getFunctions` is mocked, do we even need to return a real client here? can this just be a very basic `createTestClient()`?

Reply:
Good point, simplified `createTestClient` as much as I could